home-assignments/session1/ex1_ages_groups.py

```diff
@@ -0,0 +1,46 @@
+import json
+import yaml
+
+
+with open(r'C:\Users\HEZI LEVI\PycharmProjects\ops_assingment1\ppl_ages') as f:
+    data = json.load(f)
+
+
+
+def ppl_buckets(data):
+    ppl_ages = data['ppl_ages']
+
+    buckets = sorted(data['buckets'])
+
+    a = []
```

Comment:
The bucket list should be dynamic (and a list itself)
What will happen to your program if the list will be in the length of 6 or 10?